event/oracle/event.go

```diff
@@ -0,0 +1,103 @@
+package oracle
```

Comment:
Functions commonly used by `OracleRegistrationVote` and `OracleUpgradeVote` are gathered here.